file/src/test/java/FileProcessingUtil.java

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
Should we transform this into a test and/or include in the docs?

Reply:
This shouldn't have been there. It became `NestedTarReaderTest`.